File: Departure/Assets/DisplayScore.cs
using UnityEngine;
using System.Collections;

public class DisplayScore : MonoBehaviour {

	
	void OnGUI() 
	{
		// Displays Score
		GUI.Label(new Rect(	40, 
		                   	Screen.height - 50,
		                   	100,
		                   	50), 
		         			"Score: " + PlayerScore.score);
		//Displays Highscore
		GUI.Label(new Rect(	40, 
		                   	Screen.height - 70,
		                   	100,
		                   	50), 
		          			"High Score: " + PlayerScore.highScore);
		
		//GUI.Box (new Rect (Screen.width - 100,Screen.height - 50,100,50), "Bottom right");
	}
}

File: Unity Projects/SimpleGame1/Assets/Brain-GameMan-Network/DisplayScore.js
#pragma strict


// Displays scores.
function OnGUI () {
    // Player 1 Score
    GUI.Label (Rect(Screen.width/4, 5, 100, 20), "Player 1: ");
    GUI.Label (Rect ((Screen.width/4) + 55, 5, 100, 20), "" + ControlKingArea.p1Time);
    
    // Player 2 Score
    GUI.Label (Rect (Screen.width/2, 5, 100, 20), "Player 2: ");
    GUI.Label (Rect ((Screen.width/2) + 55, 5, 100, 20), "" + ControlKingArea.p2Time);

}
